pyproject.toml

```diff
@@ -1,16 +1,27 @@
-[tool.poetry]
+[project]
```

Comment:
one last change please:
move the `[build-system]` section here, at the top, as in the official example: https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#a-full-example

This way, we know right away what it builds with.